modules/monitoring-shiftstack-metrics.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * observability/monitoring/shiftstack-prometheus-configuration.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="monitoring-shiftstack-metrics.adoc_{context}"]
+= Available metrics for clusters that run on RHOSO
+
+To query metrics and identifying resources across the stack, there are helper metrics that establish a correlation between {rhoso-first} infrastructure resources and their representations in {product-title}.
+
+To map nodes with {rhoso} compute instances, in the metric `kube_node_info`:
```

Comment:
Correct or wrong cluster?

Reply:
correct! "nodes" here is "OpenShift nodes", and "compute instances" is "rhoso compute instances".